lib/utility/convertDateToJSFDateTime.js

```diff
@@ -0,0 +1,11 @@
+var jsforce = require('jsforce');
+
+/**
+ * Convert Date filter props to SalesForce based date fields
+ * @param {field} value
+ * @param {fieldType} salesforce type value
```

Comment:
The correct order for `@param` JSDoc annotation is type, name and optional description. Please change to something like this: `@param {string} fieldType Original type from salesforce schema`. Same applies to the first parameter.